dt-core/admin/menu/tabs/tab-workflows.php

```diff
@@ -275,9 +275,17 @@ public function content( $tab ) {
 
             $this->template( 'begin' );
 
+            ?>
+            <div id="post-body-content">
+            <?php
             $this->workflows_post_types_section( $selected_post_type );
             $this->workflows_management_section( $selected_post_type );
             $this->workflows_design_section( $selected_post_type );
+            ?>
+            </div>
+
+            <?php $this->right_column() ?>
```

Comment:
Before the right column, we need to add `$this->template('right_column');` to add the HTML for the sidebar.